be/src/exec/olap_common.h

```diff
@@ -662,6 +662,8 @@ Status OlapScanKeys::extend_scan_key(ColumnValueRange<T>& range) {
         return Status::OK;
     }
 
+    bool _has_converted = false;
```

Comment:
Only member variable's name starts with '_', so change it to `has_converted`

Reply:
OK